test/relayHub/RelayHub.test.ts

```diff
@@ -0,0 +1,1199 @@
+                        )
+                    }
+                };
+
+                deployRequest = cloneDeployRequest(sharedDeployRequestData);
+                deployRequest.request.index = nextWalletIndex.toString();
+                misbehavingVerifier =
+                    await TestDeployVerifierConfigurableMisbehavior.new();
+                deployRequest.request.index = nextWalletIndex.toString();
+                nextWalletIndex++;
+
+                relayRequestMisbehavingVerifier =
+                    cloneDeployRequest(deployRequest);
+                relayRequestMisbehavingVerifier.relayData.callVerifier =
+                    misbehavingVerifier.address;
+
+                const dataToSign = new TypedDeployRequestData(
+                    chainId,
+                    factory.address,
+                    relayRequestMisbehavingVerifier
+                );
+                signatureWithMisbehavingVerifier = getLocalEip712Signature(
+                    dataToSign,
+                    gaslessAccount.privateKey
+                );
+            });
+
+            it('Should faild a deployRequest if SmartWallet has already been initialized', async () => {
+                await relayHubInstance.stakeForAddress(relayManager, 1000, {
+                    value: ether('2'),
+                    from: relayOwner
+                });
+                await relayHubInstance.addRelayWorkers([relayWorker], {
+                    from: relayManager
+                });
+                await relayHubInstance.registerRelayServer(url, {
+                    from: relayManager
+                });
+
+                const smartWalletTemplate: SmartWalletInstance =
+                    await SmartWallet.new();
+                const senderAccount = web3.eth.accounts.create();
+
+                smartWalletTemplate.initialize(
+                    senderAccount.address,
+                    token.address,
+                    relayWorker,
+                    '0',
+                    '400000'
+                );
+                factory = await createSmartWalletFactory(smartWalletTemplate);
+
+                const calculatedAddr = await factory.getSmartWalletAddress(
+                    gaslessAccount.address,
+                    constants.ZERO_ADDRESS,
+                    relayRequestMisbehavingVerifier.request.index
+                );
+                await token.mint('1', calculatedAddr);
+
+                await assert.isRejected(
+                    relayHubInstance.deployCall(
+                        relayRequestMisbehavingVerifier,
+                        signatureWithMisbehavingVerifier,
+                        { from: relayWorker, gas, gasPrice }
+                    ),
+                    'Unable to initialize SW',
+                    'SW was deployed and initialized'
+                );
+            });
+
+            it('Should faild a deployRequest if Manager is unstaked', async () => {
```

Comment:
faild

Reply:
The word faild is misspelled and should be updated to failed.

Reply:
✅